controlplane/modules/route/internal/discovery/bird/testdata/fuzz/Fuzz_update_Decode/0aa332e51f5629d7

```diff
@@ -0,0 +1,2 @@
+go test fuzz v1
```

Comment:
I think we don't want to commit fuzzer artifacts. Why do we want to?

Reply:
Running `go test ./...` will execute the fuzz tests with these artifacts as regular test cases, which can be helpful if the bug reappears. Therefore, it seems simpler and more reliable to commit the fuzz artifacts rather than rewrite this test by hand.